### PR TITLE
Expand child place types and re-name sitemap to place list

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/templates/base.html [hunk 1]
```diff
@@ -132,7 +132,7 @@ <h6>Data Commons</h6>
               <a href="#">About Data Commons</a>
               <a href="{{ url_for('static.faq') }}">Frequently Asked Questions</a>
               <a href="http://github.com/datacommonsorg">Github Repository</a>
-              <a href="{{url_for('sitemap.index')}}">Sitemap</a>
+              <a href="{{url_for('placelist.index')}}">Place List</a>
```
Comment (beets) [inline server/templates/base.html:135]: Could you remove this link actually? I think we wanted to both rename and remove it from this list.

File: server/templates/placelist.html [hunk 2]
```diff
@@ -15,22 +15,22 @@
...
+{% set title = 'Place List' %}
 
 {% block head %}
   <link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['GAE_VERSION'])}} />
 {% endblock %}
 
 {% block content %}
   <div class="container">
-    <h1>Sitemap</h1>
+    <h1>Place List</h1>
```
Comment (beets) [inline server/templates/placelist.html:27]: Nit: maybe call this "Places in the Data Commons Graph"

File: server/templates/base.html (hunk 1 above)
Comment (shifucun) [reply server/templates/base.html]: Done

File: server/templates/placelist.html (hunk 2 above)
Comment (shifucun) [reply server/templates/placelist.html]: Done

**Pull request merged**